db/schema.rb

```diff
@@ -203,7 +204,6 @@
     t.string "admin_contact_email"
     t.string "admin_contact_phone"
     t.text "availability_description_description"
-    t.integer "bookings_school_id", null: false
```

Comment:
Is this column expected to be removed?

Reply:
Well spotted, no it wasn't. Force pushed to remove it.